root/etc/s6-overlay/s6-rc.d/init-limnoria-plugin-requirements/run

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/with-contenv bash
+# shellcheck shell=bash
+
+PLUGIN_REQUIREMENTS=$(find /config/plugins -maxdepth 2 -name 'requirements.txt')
+if [[ -n "${PLUGIN_REQUIREMENTS}" ]]; then
+    apk add --no-cache \
+        build-base \
+        cargo \
+        libffi-dev \
+        openssl-dev \
+        python3-dev
```

Comment:
Do we want to remove these and cleanup folders like .cargo after the package builds, or are people like to be adding and removing plugins without recreating the container?

Reply:
Depends how much it bloats the image I suppose. If every or almost ever user is going to be installing plugins then it probably makes sense to include the deps on the image rather than needing to download them on every startup.

Reply:
Tested a build with the dependencies kept in and the image ballooned from ~45mb to ~400mb, so I'm not going to consider that reasonable.

I don't currently have any extensions installed in my personal instance that result in generating any `~/.cache` or `~/.cargo` (I do have plugins with requirements, but none of them result in cache or cargo). Anyway, I'll add the removal of dependencies and cleanup folders to this script.

Reply:
Resolved in https://github.com/linuxserver/docker-limnoria/pull/9/commits/280b18c800a0711747e8a03fe64ba9d7a0e49c7b